var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -43,6 +43,9 @@ class Ascent(Package, CudaPackage):
             branch='develop',
             submodules=True,
             preferred=True)
+    version('0.6.0',
+            branch='v0.6.0',
```

Comment:
Versioned versions (vs. branches like `develop`) are usually listed by `tag` (i.e., `tag='v0.6.0'`).

I see a tag in the repo for this version but I'm not seeing a branch with that label.

Reply:
Good catch! Interestingly, I tried this and it worked ?? Either way, I will change to tag. Thanks!

Reply:
I'm not surprised that you could install the version -- through `git` perhaps -- but I wonder if it would've picked up the `submodules=True` option.